aux_ui/lib/screens/session_manager.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/services.dart';
```

Comment:
should prob move to lib or some folder for types / generic classes

Reply:
moved